common/test/acceptance/tests/lms/test_lms_instructor_dashboard.py

```diff
@@ -60,8 +60,8 @@ def setUp(self):
     def test_instructor_dashboard_a11y(self):
         self.instructor_dashboard_page.a11y_audit.config.set_rules({
             "ignore": [
+                'checkboxgroup',  # TODO: AC-552
```

Comment:
Dunno. Running the a11y tests triggered this failure. I ignored it and have another upcoming ticket that will resolve these ignores.